### PR TITLE
Haml 2 slim now works with ruby 1.9 syntax

### PR DESCRIPTION
**Pull request by brand-it**
Found it was a lot of work to change HAML to Slim if you have ruby 1.9
syntax. This can make things difficult to use this tool.

### CODE REVIEW

**Comment by newdark** [comment]
https://github.com/slim-template/haml2slim/pull/23 did not notice this pull request. Total should of checked first lol

File: haml2slim.gemspec [hunk 1]
```diff
@@ -22,4 +22,6 @@ Gem::Specification.new do |s|
   s.add_dependency "ruby_parser"
   s.add_development_dependency 'rake'
   s.add_development_dependency "slim", [">= 1.0.0"]
+  s.add_development_dependency 'minitest', '~> 5.8.3'
+  s.add_development_dependency 'pry'
```
Comment (orospakr) [inline haml2slim.gemspec]: you added this as a development dependency, but it appears required at runtime as well.

Comment (newdark) [inline haml2slim.gemspec]: removed the `require pry`from the converter.rb must of left it in after testing out the tool.

**Comment by newdark** [comment]
This gem still has a bug with things like `data: { something: 'test' }`. Depending on the way the haml is formated it replaces a bit funny.

**Pull request closed without merging**